### DISCUSSION TITLE
我的电脑是win11  家庭中文版23h2,原来安装过这个软件，卸载一次再也安装不成功了，提示如下图，求救怎么才能成功安装？

### DISCUSSION THREAD

**Discussion by LPLWWS** [zetaloop/ExplorerPatcher#50] [Q&A]
![Uploading 1709744399856.jpg…]()

**Comment by LPLWWS** [comment]
![1709744399856](https://github.com/zetaloop/ExplorerPatcher/assets/12853250/2f49a0f2-1435-484c-9f49-f676e884b1c0)

**Reply by zetaloop** [reply]
找找有没有ExplorerPatcher的程序文件没有删除干净的？

**Reply by LPLWWS** [reply]
我是用GeekUninstaller卸载的，应该是卸载干净了，

**Reply by zetaloop** [reply]
检查以下文件是否删除干净：
C:\Program Files\ExplorerPatcher\
C:\Windows\dxgi.dll
再用everything搜一下还有没有没删干净的dxgi.dll，**注意点开文件属性里详细信息写了ExplorerPatcher的才是EP的注入文件，没写的是系统本身的dxgi组件，不要删错了！**
